hr_contract_cae/views/hr_contract.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright 2019 Coop IT Easy
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <record id="hr_contract_view_form" model="ir.ui.view">
+        <field name="name">hr_contract_view_form</field>
+        <field name="model">hr.contract</field>
+        <field name="inherit_id" ref="hr_contract.hr_contract_view_form"/>
+        <field name="arch" type="xml">
+
+            <field name="reported_to_secretariat" position="after">
+                <field name="reason"/>
+            </field>
+
+            <field name="date_start" position="after">
+                <label for="duration"/>
+                <div class="o_row" name="duration">
+                    <field name="duration" nolabel="1"/>
```

Comment:
Could you add make compatible the different numeric fields with my widget numeric step (here is [README](https://github.com/OCA/web/pull/1362/files?short_path=f0f8534#diff-f0f853481176f087bed8f59d72b6b3f6)) ?

For example, here it would be : `<field name="duration" nolabel="1" widget="numeric_step" options="{'step': 1, 'min': 0}"/>`

Reply:
@robinkeunen, what do you think of this idea?

Reply:
@quentinDupont I'm not sure I understand what needs to be done to make it compatible. Is it just replacing line 17-23 with 
```
<field name="duration" widget="numeric_step" options="{'step': 1, 'min': 0}"/> days
```
?

Reply:
Oops nope, just line 20 with this !